### ISSUE TITLE
Suggest you modify build sequence to allow enable/disable without savebreaking.

### ISSUE THREAD

**Issue by pheonix99** [Truinto/DarkCodex#20]
Tabletop Tweaks uses a setup where all the new blueprints are built regardless of settings but are only  added to selection lists if they're toggled on. This means they don't break saves if disabled after install.

Recommend you copy this.

**Comment by Truinto** [comment]
I have tried two different approaches for this. Both attempts failed. I am sure it would be possible to purge non-existent features from save data (because I managed to do this once with a save breaking component). But it's really hard to code it in a way that it works with everything.

About TT's approach. That's basically keeping it enabled, but having it not visible. I don't like it, because if there is an incompatibility between mods, it will happen regardless of the setting. If you want it to be in the game, but not use it, you can simple not pick it during level up...

I can understand the frustration of wanting to disable a setting, but then not be able to continue with a save. But it's the same for every content mod. Non can be uninstalled from a save. I do hope that at some point we get a save sanitation for that. I honestly expected that to be in the base game.

Tl;dr I think my approach is better. I will think about it, but probably leave it like it is.

**Comment by Truinto** [comment]
Btw, thanks for the feedback.
